README.md

```diff
@@ -9,10 +9,12 @@
 The Square Point of Sale SDK lets you quickly and easily add support to your application for completing in-store payments using Square Point of Sale.
 
 ## Requirements
-* A [Square account](https://squareup.com/signup).
+* A [Square developer account](https://squareup.com/signup?v=developers).
 * Xcode 8.0 or later.
 * iOS 9 or later.
 
+It is not currently possible to process a fake credit card payment with the Point of Sale API. If you want to test processing card payments, visit [squareup.com/activate](https://www.squareup.com/activate) after you create your account to ensure your account is enabled for payment processing.
```

Comment:
Would we be able to add more information here as well to say:

We recommend charging $1 amounts to test that the payment is working. You can go into your account and refund the transaction here (insert link or documentation on transaction refunding)